tools/terraform/ecs.tf

```diff
@@ -1,22 +1,20 @@
-data "aws_region" "current" {}
```

Comment:
this appears to be some merge conflict resolution gone awry? we're deleting the alb r53 record and replacing with old commented out code?  I don't think we ever use the r53 record to the LB, but if we had osquery hosts within the VPC, it might be useful, rather than traversing the NAT gateway to the internet and coming back in.